mono/metadata/threads.c

```diff
@@ -3603,6 +3603,111 @@ get_thread_dump (MonoThreadInfo *info, gpointer ud)
 	return MonoResumeThread;
 }
 
+typedef struct QuickDumpUserData {
+	char *bufp;
+	char *maxp;
+	const MonoUnityCallstackOptions *opts;
+	size_t prefix_len;
+} QuickDumpUserData;
+
+static void
+append_quick (QuickDumpUserData *ud, const char *str)
+{
+	int len = (int)strlen (str);
+	int remain = (int)(ud->maxp - ud->bufp);
+	int copylen = len;
+	if (copylen > remain)
+		copylen = remain;
+	memcpy (ud->bufp, str, copylen);
+	ud->bufp += copylen;
+}
+
+static int
+append_parent_types (QuickDumpUserData *ud, const MonoClass *k)
```

Comment:
Is this ever used?

Reply:
Good catch, no it isn't anymore - I was using it before I had to resort to the full name thing. I'll take it out.